### PR TITLE
Bump winfsp version to 1.8 in CI

### PR DESCRIPTION
**Pull request by vxgmichel**
And migrate to github actions.

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -4,4 +4,4 @@ pre-commit
 wheel
 twine
 coverage
-pytest-cov
\ No newline at end of file
+pytest-cov
```
Comment (touilleMan) [inline requirements-dev.txt:7]: what has changed here ?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        with:
+          env_vars: PYTHON_VERSION, ARCHITECTURE
+
+      - name: Upload wheel as an artifact
+        uses: actions/upload-artifact@v2
+        with:
+          path: ./wheelhouse/*.whl
+
+  Release:
+    runs-on: windows-latest
```
Comment (touilleMan) [inline .github/workflows/ci.yml:60]: you can also make this job run on linux like you do for the Quality ;-)

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.0
+
+  Build:
+    name: Build wheel for ${{ matrix.python_version }}-${{ matrix.architecture }}
+
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python_version: [cp36, cp37, cp38]
```
Comment (touilleMan) [inline .github/workflows/ci.yml]: no 3.9 ?

**Review by touilleMan — APPROVED** [review]

File: requirements-dev.txt (hunk 1 above)
Comment (vxgmichel) [reply requirements-dev.txt:7]: Simply a missing line feed.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (vxgmichel) [reply .github/workflows/ci.yml:60]: Actually no, we have to run windows in order to build for windows :)

File: .github/workflows/ci.yml (hunk 3 above)
Comment (vxgmichel) [reply .github/workflows/ci.yml]: 49fd40a

**Pull request merged**